cmd/kops-controller/pkg/config/options.go

```diff
@@ -55,6 +55,9 @@ type ServerOptions struct {
 	SigningCAs []string `json:"signingCAs"`
 	// CertNames is the list of active certificate names.
 	CertNames []string `json:"certNames"`
+
+	// UseInstanceIDForNodeName uses the instance ID instead of the hostname for the node name.
```

Comment:
Currently only AWS supports verified nodes, but should we still say that this is AWS specific?

Reply:
GCE also has a verifier, but it doesn't have this issue of its CCM supporting two node naming conventions.